lib/ui/view/channel/channel_info/widgets/second_section.dart

```diff
@@ -4,28 +4,35 @@ import 'package:flutter/material.dart';
 import '../../../../shared/colors.dart';
 import 'textstyles.dart';
 
-class SecondSection extends StatelessWidget {
+class SecondSection extends StatefulWidget {
```

Comment:
why stateful widgets, stacked doesnt make use of stateful widget. Follow the stacked architecture process